src/test/resources/dialogmelding_dialog_notat_in_modia.xml

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<EI_fellesformat xmlns="http://www.nav.no/xml/eiff/2/"
+                 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <MsgHead xmlns="http://www.kith.no/xmlstds/msghead/2006-05-24">
+        <MsgInfo>
+            <Type DN="Notat" V="DIALOG_NOTAT" />
+            <MIGversion>v1.2 2006-05-24</MIGversion>
+            <GenDate>2019-01-16T22:51:35.5317672+01:00</GenDate>
+            <MsgId>47340D30-FE14-42B5-985F-A8FF8FFA0CA6</MsgId>
```

Comment:
Litt vanskelig når man skal teste for verdier i XMLen, men er altså denne verdien som må matche med `UserConstants.MSG_ID_IN_MODIA` for at man skal få et treff fra mocken.